builddefs/common_rules.mk

```diff
@@ -130,11 +128,7 @@ ifeq ($(ADHLNS_ENABLE),yes)
   else
     CFLAGS += -Wa,-adhlns=$(@:%.o=%.lst)
     CXXFLAGS += -Wa,-adhlns=$(@:%.o=%.lst)
-    ifeq ($(strip $(DEBUG_ENABLE)),yes)
-      ASFLAGS = -Wa,-adhlns=$(@:%.o=%.lst),-gstabs,--listing-cont-lines=100
-    else
-      ASFLAGS = -Wa,-adhlns=$(@:%.o=%.lst),--listing-cont-lines=100
-    endif
+	ASFLAGS = -Wa,-adhlns=$(@:%.o=%.lst),--listing-cont-lines=100
```

Comment:
Does the `=` here not override the previous `ASFLAGS += -ggdb3`? or would we consider the two options `ADHLNS_ENABLE` and `DEBUG_ENABLE` mutually exclusive?

Reply:
Whoops, no that is just an oversight on my part. Corrected.